yb-voyager/cmd/importDataStatusCommand.go

```diff
@@ -16,53 +16,111 @@ limitations under the License.
 package cmd
 
 import (
+	"encoding/json"
 	"errors"
 	"fmt"
 	"os"
 	"path"
 	"path/filepath"
 	"sort"
 	"strings"
+	"syscall"
 
 	"github.com/fatih/color"
+	"github.com/google/uuid"
 	"github.com/gosuri/uitable"
 	"github.com/spf13/cobra"
+	"golang.org/x/term"
 
 	"github.com/yugabyte/yb-voyager/yb-voyager/src/datafile"
 	"github.com/yugabyte/yb-voyager/yb-voyager/src/datastore"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/tgtdb"
 	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
 )
+var ffDBPassword string 
+
+func validateFallforwardPassword(cmd *cobra.Command) {
+	if cmd.Flags().Changed("ff-db-password") {
+		return
+	}
+	if os.Getenv("FF_DB_PASSWORD") != "" {
+		tconf.Password = os.Getenv("FF_DB_PASSWORD")
+		return
+	}
+	fmt.Print("Password to connect to fall-forward DB:")
```

Comment:
In this message, also mention that user can also set password in an env var (give the name of env var).